_episodes/40.md

```diff
@@ -0,0 +1,479 @@
+While Trino was able to read tables with these migrations applied by Spark, this
+feature allows Trino to write these changes without the need of another query
+engine.
+* The following three table maintenence commands were released in versions 
+[369]({{site.url}}docs/current/release/release-369.html#iceberg-connector) and
+[378]({{site.url}}docs/current/release/release-378.html#iceberg-connector).
+    * [`optimize`]({{site.url}}/docs/current/connector/iceberg.html#optimize)
+([PR](https://github.com/trinodb/trino/pull/10497)) which is the equivalent to
+the Spark SQL 
+[rewrite_data_files](https://iceberg.apache.org/docs/latest/spark-procedures/#rewrite_data_files)
+and is used for rewriting the active content of the specified table so that it 
+is merged into fewer but larger files. This operation improves read performance
+by avoiding the small-files problem in big data.
+    * [`expire_snapshots`]({{site.url}}/docs/current/connector/iceberg.html#expire-snapshots)
+([PR](https://github.com/trinodb/trino/pull/10810)) removes all snapshots and
+all related metadata and data files. Regularly expiring snapshots is recommended
+to delete data files that are no longer needed, and to keep the size of table
+metadata small.
+    * [`remove_orphan_files`]({{site.url}}/docs/current/connector/iceberg.html#remove-orphan-files)
+    ([PR](https://github.com/trinodb/trino/pull/10810)) command removes all 
+files from table’s data directory which are not linked from metadata files and 
+that are older than the value of retention_threshold parameter.
+* Iceberg v2 support ([PR1](https://github.com/trinodb/trino/pull/11880), [PR2](https://github.com/trinodb/trino/pull/12351), [PR3](https://github.com/trinodb/trino/pull/12749), [PR4](https://github.com/trinodb/trino/pull/11642), [PR5](https://github.com/trinodb/trino/pull/9881), and many more...)
+
+
+Almost every release, there is some sort of performance improvement around
+many use cases involving [planning](https://github.com/trinodb/trino/pull/13636),
+[pushdown](https://github.com/trinodb/trino/pull/13395), 
+[avoiding excessive scanning](https://github.com/trinodb/trino/pull/7367), and
+much more.  A lot of the common use cases have been covered but please 
+[open an issue](https://github.com/trinodb/trino/issues/new) if you experience
+any slow or abnormal behavior in the Iceberg connector.
+
+If you want all the latest features we described here and keep up with all these
+performance improvements, it's important to keep up with the latest Trino 
+version. Of all the connectors, the connectors that support data lake, and
+object storage are the most common changes we're seeing.
+
+## PR 13111: Scale table writers per task based on throughput
+
+This [PR of the episode](https://github.com/trinodb/trino/pull/13111) was 
+contributed by Gaurav Sehgal ([@gaurav8297](https://github.com/gaurav8297)) to 
+enable Trino to automatically scale Iceberg writers. This PR aims to the number
```

Comment:
really .. I dont think so .. this is for scaling any writers right?

Reply:
yup yup yup